tests/unit/amo/components/TestSearchResult.js

```diff
@@ -130,17 +130,25 @@ describe(__filename, () => {
   });
 
   it('renders the user count', () => {
-    const root = render();
+    const root = render({ addon: { ...baseAddon, average_daily_users: 6233 } });
 
-    expect(root.find('.SearchResult-users')).toIncludeText('5,253 users');
+    expect(root.find('.SearchResult-users')).toIncludeText('6,233 users');
   });
 
   it('localises the user count', () => {
-    const root = render({ lang: 'fr' });
+    const root = render({
+      addon: { ...baseAddon, average_daily_users: 6233 },
+      lang: 'fr',
+    });
 
-    // `\u202F` is a narrow non-breaking space, see:
-    // https://www.fileformat.info/info/unicode/char/202f/index.htm
-    expect(root.find('.SearchResult-users-text')).toIncludeText('5\u202F253');
+    expect(
+      root
+        .find('.SearchResult-users-text')
+        .text()
+        // This handles `\u202F` which is a narrow non-breaking space, see:
+        // https://www.fileformat.info/info/unicode/char/202f/index.htm
```

Comment:
This comment does not seem super accurate anymore. Maybe we can say that the non-breaking space character varies depending on the platform, node version, alignment of planets?

Reply:
I changed the comment from the original. It's now saying that the regex "handles" the non-breaking space, just as a way to communicate why the regex exists.

Reply:
but, yeah, I can mention that the character varies so that someone isn't tempted to change the code!